src/main/java/org/zaproxy/zest/core/v1/ZestClientElementMouseOver.java

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+package org.zaproxy.zest.core.v1;
+
+import org.openqa.selenium.interactions.Actions;
+
+public class ZestClientElementMouseOver extends ZestClientElement {
```

Comment:
Worth documenting that the element must be in view already (the "scroll to" statement would make that easier, especially when being added by the user).

Reply:
where should I mention this thing, inside this file only?

Reply:
Yes, in JavaDoc of the class should be enough.

Reply:
okk thanks will do it as soon as possible